README.md

```diff
@@ -16,7 +16,16 @@ MRTK-Unity is a Microsoft-driven project that provides a set of components and f
 
 # Getting started with MRTK
 
-| [![Getting Started and Documentation](Documentation/Images/MRTK_Icon_GettingStarted.png)](https://docs.microsoft.com/windows/mixed-reality/unity-development-overview)<br/>[Getting Started](https://docs.microsoft.com/windows/mixed-reality/unity-development-overview)| [![Getting Started](Documentation/Images/MRTK_Icon_ArchitectureOverview.png)](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Architecture/Overview.html)<br/>[MRTK Overview](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Architecture/Overview.html)| [![Feature Guides](Documentation/Images/MRTK_Icon_FeatureGuides.png)](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/README_Button.html)<br/>[Feature Guides](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/README_Button.html)| [![API Reference](Documentation/Images/MRTK_Icon_APIReference.png)](https://microsoft.github.io/MixedRealityToolkit-Unity/api/Microsoft.MixedReality.Toolkit.html)<br/>[API Reference](https://microsoft.github.io/MixedRealityToolkit-Unity/api/Microsoft.MixedReality.Toolkit.html)|
+If you're new to MRTK or Mixed Reality development in Unity, **we recommend you start at the beginning of our** [Unity development journey](https://docs.microsoft.com/windows/mixed-reality/unity-development-overview?tabs=mrtk%2Chl2) in the Microsoft Docs. The Unity development journey is specifically tailored to walk new developers through the installation, core concepts, and usage of MRTK. 
+
+| CAUTION: The Unity development journey currently uses **MRTK version 2.4.0** and **Unity 2019.4**. |
```

Comment:
@hferrone, any particular reason you've chosen to use a caution tag here as opposed to an Important tag?

Reply:
Nothing in particular, I can change it if you like.

Reply:
Yes, can you please change it? Thanks!